apps/mull-ui/src/utilities.ts

```diff
@@ -145,3 +145,28 @@ export const sortEventsByDate = (events: Partial<ISerializedEvent>[]) => {
         new Date(event1.startDate).valueOf() - new Date(event2.startDate).valueOf()
     );
 };
+
+/**
+ * converts a float between 0 to 24 to a readable time format
+ * 23.75 -> 23:45
+ * @param minutes
+ */
+export const floatToHHMM = (minutes: number): string => {
+  const hour = Math.floor(Math.abs(minutes));
+  const min = Math.floor((Math.abs(minutes) * 60) % 60);
+  return (hour < 10 ? '' : '') + hour + ':' + (min < 10 ? '0' : '') + min;
```

Comment:
Just a question, why is the ternary operator always returning `''`  for the hour < 10 statement?